drivers/volume/portworx/portworx.go

```diff
@@ -5232,7 +5232,8 @@ func (d *portworx) ValidateDriver(endpointVersion string, autoUpdateComponents b
 		}
 
 		// Validate StorageCluster
-		if err = optest.ValidateStorageCluster(imageList, newStc, validateStorageClusterTimeout, defaultRetryInterval, true); err != nil {
+		storageClusterValidateTimeout := time.Duration(len(node.GetStorageDriverNodes())*9) * time.Minute
```

Comment:
Why 9 minutes or 10 minutes for each nodes? How did you reach on that calculation ?